purchases/build.gradle

```diff
@@ -31,55 +32,23 @@ dependencies {
     testImplementation 'org.assertj:assertj-core:3.13.2'
 }
 
-apply plugin: 'com.vanniktech.maven.publish'
-apply plugin: 'org.jetbrains.dokka'
-
-dokka {
-    outputFormat = 'html'
-    outputDirectory = "$rootDir/docs"
-
-    configuration {
-        moduleName = 'purchases'
-        includes = ['README.md']
-        reportUndocumented = true
-        includeNonPublic = false
-        skipDeprecated = true
-        // Allows linking to documentation of the project's dependencies (generated with Javadoc or Dokka)
-        // Repeat for multiple links
-        externalDocumentationLink {
```

Comment:
did we confirm the trailing slash is not required?

Reply:
I actually never got this to work. Does this work in the new version? I was looking for an example but I can't find it. I think we don't have any public link to the billing client anymore and everything is in the `google` module

Reply:
happy to remove it, I kept it around for the sake of having everything that we had before

Reply:
Agreed, I would keep it the same it was before